scripts/pre-commit.sh

```diff
@@ -2,4 +2,3 @@
 set -e
 
 ./scripts/ban.sh
-npm run test
```

Comment:
Removed it as it takes "too much time" between two commits and as we have CI anyway